### PR TITLE
Remove heroku startup script

### PR DESCRIPTION
**Pull request by kevindew**
This has now been superseded with the review app system provided by
Heroku. I'm removing this to simplify the application so there are less
choices for developers when working with this and as part of work
cleaning up legacy cruft built up in this repo.

I've also updated the fact-check documentation to remove references to
this script and remove out-of-date information.

### CODE REVIEW

File: doc/smart-answer-flow-development/factcheck.md [hunk 1]
```diff
@@ -1,35 +1,21 @@
...
 
-When making bigger changes that need to be tested or fact-checked before they are deployed to GOV.UK you'll need to deploy the branch with changes to Heroku.
+When making bigger changes that need to be tested or fact-checked before they
+are deployed to GOV.UK you can make use of [Heroku Review apps][].
+These are created automatically when a new PR is opened and will also be
+automatically linked to from the PR. They will also update on each new commit
+to the PR.
 
-## Deploying to Heroku
+## Displaying rates for a specific date
```
Comment (benthorner) [inline doc/smart-answer-flow-development/factcheck.md]: (I still have no idea what this is.)

**Comment by kevindew** [comment]
@benthorner sure, I've dropped a couple of changes in to cover those things you flagged.

File: doc/smart-answer-flow-development/fact-check.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+The master branch of Smart Answers is deployed to Heroku with each update and
+this allows previewing draft Smart Answers. This is available at
+https://smart-answers-preview.herokuapp.com/.
+
+## Displaying rates for a specific date
+
+Smart Answers are often used to illustrate certain costs or benefits applicable
+on a particular date. These are determined by looking up the correct rates (in
+[`lib/data/rates`](../../lib/data/rates)) to display for the current date.
+When testing upcoming rate changes to rates you may want to simulate how
```
Comment (benthorner) [inline doc/smart-answer-flow-development/fact-check.md:18]:
```suggestion
When testing upcoming changes to rates you may want to simulate how
```

Comment (kevindew) [reply doc/smart-answer-flow-development/fact-check.md]: Thanks 🤦

**Pull request merged**